src/components/Button/Button.tsx

```diff
@@ -12,9 +13,13 @@ const Button: React.FC<ButtonProps> = ({ onClick, children, disabled }) => {
 
```

Comment:
так не надо импортировать, надо использовать только один тип импортов, и вынести за пределы функции
```suggestion
import cn from 'classnames';
```

Reply:
но кстати здесь эта либа ненужна, та либа нужна только тогда, когда у нас стиль надо добавить по какому-то условию